### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by mlila**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Hello World 
+
+
+## Intro
```
Comment (MichaelXXI) [inline README.md]: @mlila No intro - > Introduction Yes

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+It is a basic application where you can see a beautifull message "HELLO WORLD" with yellow backgroud :)
+
+## Functional Description
+
+TODO describe functionality
```
Comment (MichaelXXI) [inline README.md]: @mlila Finish TODO

Comment (MichaelXXI) [inline README.md]: @mlila Finish TODO

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+## Technical Description
+
+TODOD description technical descriptio
```
Comment (MichaelXXI) [inline README.md]: @mlila Finis TODO

File: README.md (hunk 1 above)
Comment (mlila) [reply README.md]: no @MichaelXXI  I think "Intro" is enough clear, I won't fix it.

**Pull request merged**